WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -4,8 +4,20 @@ class SyncMailingListsJob < ApplicationJob
   queue_as :default
 
   def perform
+    GsuiteMailingLists.sync_group("candidatedelegates@worldcubeassociation.org", User.where(delegate_status: "candidate_delegate"))
     GsuiteMailingLists.sync_group("delegates@worldcubeassociation.org", User.delegates)
+    GsuiteMailingLists.sync_group("seniors@worldcubeassociation.org", User.where(delegate_status: "senior_delegate"))
```

Comment:
Could you add a scope to the Users model for this and candidate delegates (if we don't already have them). I'd prefer that these strings not leak out of the user's model.

Reply:
Sure thing!